### PR TITLE
draft of github action for plagiarism check

### PR DESCRIPTION
**Pull request by ninjaginja**
This is a draft for a Github action that uses the Unicheck api to automate plagiarism checks upon new/changes to PRs

### CODE REVIEW

**Comment by ninjaginja** [comment]
[index.pdf](https://github.com/section-io/engineering-education/files/5103344/index.pdf)

**Comment by ninjaginja** [comment]
Using the Unicheck API, I was able to successfully generate a report from a .pdf input (converted from .md file). Attached are the command sequence, along with the outputted .pdf report.
[Unicheck-export-report-commands.txt](https://github.com/section-io/engineering-education/files/5103459/Unicheck-export-report-commands.txt)

[test.pdf](https://github.com/section-io/engineering-education/files/5103445/test.pdf)

**Comment by ninjaginja** [comment]
@auxesis I made some progress on this after working through the Unicheck API outside of GH Actions (see comment above). I've updated the .yml file (w/ commented out questions/steps).

File: .github/workflows.disabled/plagiarism-checker.yml [hunk 1]
```diff
@@ -2,90 +2,121 @@
...
+        -d '{
+          "data": {
+          "type" :"similarity-check-report-export",
+          "attributes":{
+            "format" :"pdf",
+            "locale_code" :"EN"
+              }
+            }
+          }'
+# Need to store id provided in response for use in next step
```
Comment (auxesis) [inline .github/workflows.disabled/plagiarism-checker.yml:104]: If you write the output of the curl to a file with `-o`, you could grep for it, and set it as a step variable with `echo -n "::set-output name=similarity_check_report_export_id::"`

File: .github/workflows.disabled/plagiarism-checker.yml [hunk 2]
```diff
@@ -94,6 +125,6 @@ jobs:
             issue_number: context.payload.pull_request.number,
             owner: context.repo.owner,
             repo: context.repo.repo,
-            body: "Here's a screenshot of the article (/engineering-education/${{ matrix.article_slug }}/):\n\n![screenshot](${{ steps.upload-screenshot.outputs.img_url }})"
+            body: "Here's your plagiarism check report (/engineering-education/${{ matrix.article_slug }}/):\n\n![plagiarism-check](${{ steps.output_report.outputs.${{ id }}.pdf }})"
```
Comment (auxesis) [inline .github/workflows.disabled/plagiarism-checker.yml:128]: If I've got this right, you're going to store the report PDF as an artifact on the workflow, and then link to it from the PR?

That's pretty cool!

**Comment by ninjaginja** [comment]
Closing this for now

**Pull request closed without merging**